src/data_manager/collectors/scrapers/integrations/sso_scraper.py

```diff
@@ -264,6 +264,25 @@ def authenticate_and_navigate(self, url):
         except Exception as e:
             logger.warning(f"Error during authentication: {e}")
             return None
+
+    def authenticate(self, url):
```

Comment:
if I understand correctly, the SSOScraper now provides the WebScraper with the authentication, and nothing else, right? Since now the WebScraper does everything the SSOScraper could do. Can we then remove the SSOScraper entirely, and just have a SSOAuth, or something of that sort? In short, there should only be one collector to scrape websites, the SSOAuth should just be a base class for the different specific auths like CERNSSOScraper. If you have a better proposal, let me know